src/components/cards/oodIconCard.js

```diff
@@ -1,57 +1,56 @@
 import React from 'react';
 import SbEditable from 'storyblok-react';
-import Heading from '../partials/heading';
 import SbLink from '../partials/sbLink';
+import cx from 'classnames';
 
 const OodIconCard = (props) => {
   return (
     <SbEditable content={props.blok}>
       <article
-        className={`ood-icon-card
-             ${props.blok.backgroundColor !== 'white' ? 'su-text-white' : ''}
-             su-text-align-${props.blok.contentAlign}`}
+        className={cx(
+          'ood-icon-card',
+          props.blok.backgroundColor !== 'white' ? 'su-text-white' : '',
+          `su-text-align-${props.blok.contentAlign}`
+        )}
       >
         <SbLink
           link={props.blok.link}
-          classes={`ood-icon-card__link su-bg-${props.blok.backgroundColor}
-            ${
-              props.blok.backgroundColor === 'white'
-                ? 'su-border-color-black-10'
-                : `su-border-color-${props.blok.backgroundColor}`
-            }`}
+          classes={cx(
+            'ood-icon-card__link',
+            `su-bg-${props.blok.backgroundColor}`,
+            props.blok.backgroundColor === 'white'
+              ? 'su-border-color-black-10'
+              : `su-border-color-${props.blok.backgroundColor}`
+          )}
         >
           <div className="ood-icon-card__media">
             <span
               aria-hidden="true"
-              className={`ood-icon-card__icon
-                  ${
-                    props.blok.iconStyle
-                      ? props.blok.iconStyle
-                      : props.blok.icon.type
-                  }
-                  ${
-                    props.blok.extraIcon
-                      ? `fa-${props.blok.extraIcon}`
-                      : props.blok.icon.icon
-                  }
-                  ${
-                    props.blok.backgroundColor !== 'white'
-                      ? 'su-text-white'
-                      : 'su-text-digital-red'
-                  }`}
+              className={cx(
+                'ood-icon-card__icon',
+                props.blok.iconStyle
+                  ? props.blok.iconStyle
+                  : props.blok.icon.type,
+                props.blok.extraIcon
+                  ? `fa-${props.blok.extraIcon}`
+                  : props.blok.icon.icon,
+                props.blok.backgroundColor !== 'white'
+                  ? 'su-text-white'
+                  : 'su-text-digital-red'
+              )}
             />
           </div>
           <div className="ood-icon-card__contents">
-            <Heading
-              level={props.blok.headingLevel}
-              defaultLevel={'h3'}
-              classes={'ood-icon-card__headline su-hocus-underline su-mb-none'}
-              weight={'semibold'}
-              external={props.blok.link.linktype === 'url'}
```

Comment:
I got rid of the SR text (external link) associated with the icon cards. Since there is no icon associated with it (even on the live site). The most current SODA preference is that we don't add more info for screen reader users if there's nothing visually visible associated with it. Since there is no icon when the link is external, there's no reason to add the screen reader text either.